docs/undergrad.yaml

```diff
@@ -336,11 +336,11 @@ Single Variable Real Analysis:
     inverse trigonometric functions: 'real.arcsin'
     inverse hyperbolic trigonometric functions: 'real.arsinh'
   Integration:
-    integral over a segment of piecewise continuous functions: ''
+    integral over a segment of piecewise continuous functions: 'interval_integral'
     antiderivatives: 'https://en.wikipedia.org/wiki/Antiderivative'
-    Riemann sums: 'https://en.wikipedia.org/wiki/Riemann_sum'
+    Riemann sums: 'box_integral.integral'
     antiderivative of a continuous function: 'interval_integral.integral_has_strict_deriv_at_of_tendsto_ae_right'
-    change of variable: ''
+    change of variable: `lintegral_image_eq_lintegral_abs_det_fderiv_mul`, `integral_image_eq_integral_abs_det_fderiv_smul`
```

Comment:
The result you're linking is the multivariable version, which belongs further down in section 10.  I think there is a 1-variable version somewhere in `interval_integral`, though, you can look it up.